napari/_qt/dialogs/qt_notification.py

```diff
@@ -25,8 +26,11 @@
 )
 from superqt import QElidingLabel, ensure_main_thread
 
+from ...settings import get_settings
 from ...utils.notifications import Notification, NotificationSeverity
+from ...utils.theme import get_theme
 from ...utils.translations import trans
+from ..pyhon_syntax_highlight import Pylighter
```

Comment:
Typo: rename to `python_syntax_highlight`?

Reply:
ass one could pass language I gange it to `code_syntax_highlight`. Maybe it could be part of the public API for plugin creators? or migrated to superqt (after 0.4.16 release)?

Reply:
Yeah that would be good!

Reply:
@tlambert03 What did you think about this. Did you prefer to have this as part of public API, or as part of superqt (there will be pygments dependency, maybe optional)?

Reply:
I like it,  happy for it to go into superqt.  fine if that happens later.  looks like pygments has no dependencies so probably fine just to add it directly

Reply:
It may take time because next to code there should be documentation.